src/easylogging++.h

```diff
@@ -233,7 +231,9 @@ ELPP_INTERNAL_DEBUGGING_OUT_INFO << ELPP_INTERNAL_DEBUGGING_MSG(internalInfoStre
 #  define STRCPY(a, b, len) strcpy(a, b)
 #endif
 // Compiler specific support evaluations
-#if ((!ELPP_MINGW && !ELPP_COMPILER_CLANG) || defined(ELPP_FORCE_USE_STD_THREAD))
```

Comment:
Sure. Do you have line breaking rules?—Should I break at the 80th column, or should I not break in a paragraph at all?

I ask this because line breaking seems a bit inconsistent in this file.